sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/main/java/com/azure/monitor/opentelemetry/AzureMonitorExporterBuilder.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.monitor.opentelemetry.exporter;
+package com.azure.monitor.opentelemetry;
```

Comment:
I think we probably want to match the artifact name here, e.g.

```suggestion
package com.azure.monitor.opentelemetry.autoconfigure;
```